scripts/test

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
This script should be removed until we add it via https://github.com/azavea/raster-foundry/issues/1472.